docs/DeveloperGuide.md

```diff
@@ -164,7 +164,7 @@ and only added on later to the contact with the `co NAME c/COMPANY_NAME` command
 
 The adding of a company attribute to a contact follows the following activity diagram.
 
-<puml src="diagrams/CompanyCommandActivityDiagram.puml" width="574" />
+<puml src="diagrams/CompanyCommandActivityDiagram.puml" alt="CompanyCommandActivityDiagram"/>
```

Comment:
im trying to change it to match the deletesequencediagram format because it currently looks too small in the website, so lol dk probably need to change and try to see how it looks like